http-api/src/lib.rs

```diff
@@ -78,6 +72,38 @@ impl Context {
 
         Ok(())
     }
+
+    /// Populate a map between ssh fingerprints and their peer identities
+    fn populate_fingerprints(
+        &self,
+        map: &mut HashMap<String, (PeerId, Person)>,
+    ) -> Result<(), Error> {
+        use librad::git::identities::SomeIdentity;
+
+        let storage = ReadOnly::open(&self.paths).map_err(Error::from)?;
+        let identities = identities::any::list(&storage)?;
+        for identity in identities.flatten() {
+            if let SomeIdentity::Person(person) = identity {
+                for pk in person.delegations().iter() {
+                    let peer_id = &PeerId::from(*pk);
+                    let fp = keys::to_ssh_fingerprint(peer_id).ok();
+
+                    if let std::collections::hash_map::Entry::Vacant(e) =
+                        map.entry(fp.unwrap_or_default())
```

Comment:
Using unwrap_or_default here means that if `to_ssh_fingerprint` fails, we will store an entry under the empty string `""`, which is probably not what we want!

Since this shouldn't fail unless we have a bug, I'd unwrap the result like this: 

```rust
                     let fp = keys::to_ssh_fingerprint(peer_id).expect("Conversion cannot fail");
                     ```